src/mono/netcore/System.Private.CoreLib/src/ILLink/System.Private.CoreLib.xml

```diff
@@ -753,71 +753,10 @@
 			<method signature="System.Void .ctor()" />
```

Comment:
Could you also rename the file to `ILLink.Descriptors.xml` ?

Reply:
I was going to do that as part of https://github.com/dotnet/runtime/issues/37651.